addons/payment_transfer/models/payment_acquirer.py

```diff
@@ -27,6 +27,7 @@ def _compute_view_configuration_fields(self):
             'show_cancel_msg': False,
         })
 
+    # Would be nice to be able to remove this crap honestly
```

Comment:
![do-or-do-not-there-is-no-todo](https://user-images.githubusercontent.com/15896813/185178713-6929552f-5fba-4671-a42b-d9e015086635.jpeg)